pydoctor/driver.py

```diff
@@ -330,7 +330,6 @@ def main(args=sys.argv[1:]):
                 writerclass.__name__))
 
             writer = writerclass(options.htmloutput)
-            writer.system = system
```

Comment:
?

Reply:
ah it's redundant

Reply:
Unfortunately GitHub's review system doesn't do a great job presenting details from commit comments: you either have to review individual commits or you'll get the total change set without any context. It would be useful if they had some kind of `git blame` integration to show which commit is responsible for each changed line.